### PR TITLE
TXpipe

### PR DESCRIPTION
**Pull request by chihway**
This PR makes some very small changes (column names and comments) so that TXPipe can call and run it.

### CODE REVIEW

File: flexzpipe/FlexZPipe.py [hunk 1]
```diff
@@ -25,26 +25,25 @@ class FlexZPipe(PipelineStage):
...
         "odds_int": 0.99445, #number of sigma_intrins to integrate +/- around peak         
         # note that 1.95993 is the number of sigma you get for old "ODDS" =0.95            
-        #in old BPZ, 0.68 is 0.99445                                                       
-        #"point_estimate": "mode",  # mean, mode, or median          
+        #in old BPZ, 0.68 is 0.99445                                                           
         "has_redshift": True, #does the test file have redshift?
         #if so, read in and append to output file.
         "nz": 300, #Number of grid points that FZboost will calculate   
-        "model_picklefile": "/global/homes/c/chihway/FlexZPipe/flexcode_model_sqderr.pkl", #the pickle file containing the trained  flexzbooxt model.
+        "model_picklefile": "data/example/inputs/flexcode_model_sqderr.pkl",
```
Comment (sschmidt23) [inline flexzpipe/FlexZPipe.py]: The "flexcode_model_sqderr" filename was a temporary name during testing last week where I added 'objective':'reg:squarederror' to the regression_params dictionary in the config.yaml file (xgboost renamed the parameter 'reg:linear' to 'reg:squarederror' which can cause a ton of deprecation warnings if the parameter is not specified).   As the name is not super intuitive we may want to just rename the trained pickle model file back to flexcode_model.pkl.

File: flexzpipe/FlexZPipe.py [hunk 2]
```diff
@@ -25,26 +25,25 @@ class FlexZPipe(PipelineStage):
...
+        "model_picklefile": "data/example/inputs/flexcode_model_sqderr.pkl", 
+        #the pickle file containing the trained  flexzbooxt model.
         "metacal_fluxes": False, #switch for whether or not to run metacal suffices
     }
 
     def run(self):
 
         starttime = time.time()
+
+        # not sure what the two lines below are for...
```
Comment (sschmidt23) [inline flexzpipe/FlexZPipe.py]: At least several months ago, not setting the environment variable HDF5_USE_FILE_LOCKING="FALSE" led to errors when using h5py-parallel, though I have not checked whether any changes have rendered it unnecessary.
Similarly, the CECI_SETUP was required by older versions of ceci

**Review by sschmidt23 — APPROVED** [review]
Looks good, I added a comment about the naming of flexcode_model_sqderr, we should think about shortening that back to just flexcode_model.pkl, as the "sqderr" was a temporary test while debugging by me.  The flexcode_model_sqderr.pkl file should work with <1.0 and >=1.0 versions of xgboost, so I don't think there are any issues with just renaming it to flex code_model.pkl

File: flexzpipe/FlexZPipe.py (hunk 2 above)
Comment (joezuntz) [reply flexzpipe/FlexZPipe.py]: `CECI_SETUP` will be ignored in newer ceci versions.  `HDF5_USE_FILE_LOCKING` is still needed by lots of things - it's set in the shifter images, which are now the best way of running I think.

Maybe the best solution here would be to add an `env` section to ceci so you can set things like this manually.

Comment (joezuntz) [reply flexzpipe/FlexZPipe.py]: I've updated the comment here.

File: flexzpipe/FlexZPipe.py (hunk 1 above)
Comment (joezuntz) [reply flexzpipe/FlexZPipe.py]: I've renamed this file.

Comment (joezuntz) [reply flexzpipe/FlexZPipe.py]: oops, looks like this hits the file size limit, so I can't push.  Will change it to be hosted at NERSC and add a note to download it.

Comment (joezuntz) [reply flexzpipe/FlexZPipe.py]: Okay, done now.

**Comment by sschmidt23** [comment]
Looks good to me, merging.

**Pull request merged**